### DISCUSSION TITLE
Making Turrets Shoot Instanly.

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#98] [Ideas]
I really have no idea if this is possible, possibly removing the windup time for the turret to shoot?

**Comment by winstxnhdw** [comment]
I might be not be able to get it to shoot instantly, but there may be a way to get it to shoot faster. It doesn't look simple in my head so I'll put this on the backlog.

**Reply by yujispin990** [reply]
Yay! thank you for your consideration!
